### PR TITLE
istio api evolution blog post

### PR DESCRIPTION
**Pull request by crcsmnky**
A blog post from Google about the evolution of Istio's APIs. 

[ ] Configuration Infrastructure
[X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/blog/2019/evolving-istios-apis/index.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+Over the course of the next several releases we will share our progress as we strengthen the alignment between Istio’s APIs and the roles of Istio users.
+
+## Composability and abstractions
+
+Istio and Kubernetes often go together, but Istio is much more than an add-on to Kubernetes – it is as much a _platform_ as Kubernetes is. Istio aims to provide infrastructure, and surface the capabilities you need in a powerful service mesh. For example, there are platform-as-a-service offerings that use Kubernetes as their foundation, and build on Kubernetes’ composability to provide a subset of APIs to application developers. 
+
+The number of objects that must be configured to deploy applications is a concrete example of Kubernetes’ composability. By our count, at least 10 objects need to be configured:
+
+* Namespace
```
Comment (rcaballeromx) [inline content/blog/2019/evolving-istios-apis/index.md]: Please use dashes for lists:

```suggestion
- Namespace
```

File: content/blog/2019/evolving-istios-apis/index.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+* Service
```
Comment (rcaballeromx) [inline content/blog/2019/evolving-istios-apis/index.md]:
```suggestion
- Service
```

File: content/blog/2019/evolving-istios-apis/index.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+AutoTrader UK has one of our favorite examples of a custom platform built on Istio. In [an interview with the Kubernetes Podcast from Google](https://kubernetespodcast.com/episode/052-autotrader/), Russel Warman and Karl Stoney describe their Kubernetes-based delivery platform, with [cost dashboards using Prometheus and Grafana](https://karlstoney.com/2018/07/07/managing-your-costs-on-kubernetes/). With minimal effort, they added configuration options to determine what their developers want configured on the network, and it now manages the Istio objects required to make that happen. There are countless other platforms being built in enterprise and cloud-native companies: some designed to replace a web of company-specific custom scripts, and some aimed to be a general-purpose public tool. As more companies start to talk about their tooling publicly, we'll bring their stories to this blog.
+
+## What’s coming next
+
+Some areas of improvement that we’re working on for upcoming releases include: 
+
+* Installation profiles to setup standard patterns for ingress and egress, with the Istio operator
```
Comment (rcaballeromx) [inline content/blog/2019/evolving-istios-apis/index.md]: Please change all `*` to `-` for lists.

```suggestion
- Installation profiles to setup standard patterns for ingress and egress, with the Istio operator
```

File: content/blog/2019/evolving-istios-apis/index.md (hunk 1 above)
Comment (crcsmnky) [reply content/blog/2019/evolving-istios-apis/index.md]: fixed

File: content/blog/2019/evolving-istios-apis/index.md (hunk 2 above)
Comment (crcsmnky) [reply content/blog/2019/evolving-istios-apis/index.md]: fixed

File: content/blog/2019/evolving-istios-apis/index.md (hunk 3 above)
Comment (crcsmnky) [reply content/blog/2019/evolving-istios-apis/index.md]: fixed

**Comment by crcsmnky** [comment]
@rcaballeromx please retest, errors should all be addressed now.

**Pull request merged**